client/asset/dcr/rpcwallet.go

```diff
@@ -110,6 +110,17 @@ func (cc *combinedClient) ValidateAddress(ctx context.Context, address stdaddr.A
 	return cc.walletClient.ValidateAddress(ctx, address)
 }
 
+// AbandonTransaction calls the abandontransaction RPC method.
```

Comment:
Can add a line to show that this implements the TxAbandoner interface and also checks that,

```
var _ asset.TxAbandoner = (*combinedClient)(nil)
```